pkg/agent/openflow/packetin.go

```diff
@@ -53,6 +56,9 @@ const (
 	// PacketInReasonMC shares PacketInReasonNP for IGMP packet_in message. This is because OVS "controller" action
 	// only correctly supports reason 0 or 1. Change to another value after the OVS action is corrected.
 	PacketInReasonMC = PacketInReasonNP
+	// PacketInReasonSvcReject shares PacketInReasonNP for IGMP packet_in message. This is because OVS "controller" action
```

Comment:
```suggestion
	// PacketInReasonSvcReject shares PacketInReasonNP to process the Service packet not matching any Endpoints within packet_in message. This is because OVS "controller" action
```

Reply:
Thanks.